### DISCUSSION TITLE
An established connection was aborted by the software in your host machine

### DISCUSSION THREAD

**Discussion by Sliller** [v2fly/v2ray-core#1686] [Q&A]
[Warning] [1004404892] app/proxyman/inbound: connection ends > proxy/http: connection ends > proxy/http: failed to write response > write tcp 127.0.0.1:10809->127.0.0.1:54088: wsasend: An established connection was aborted by the software in your host machine
不知道是什么原因导致的，无法下载资源，访问网页也很慢。

**Comment by FuGuiLiu** [comment]
把telegram 卸载了 装一个portable 版本

**Reply by Hommit** [reply]
Thanks, thats it
Actually, use can just set it "Don't use proxy" in advanced settings and error spam stops
